packages/salesforcedx-vscode-apex/test/jest/views/settings/configurations.test.ts

```diff
@@ -0,0 +1,19 @@
+import * as vscode from 'vscode';
```

Comment:
@mingxuanzhangsfdx This test is checking if `vscode.workspace.getConfiguration()` is working, which we have to assume is true, given it is provided by a third party. What is important is how we are using vscode settings to attain the desired behavior.

Please consider writing a test that answers the quest "does the code that modifies the server initialization options properly apply the custom settings `salesforcedx-vscode-apex.wait-init-jobs`?"

Reply:
Does this test deserve to implement and look good to check if vscode.workspace.getConfiguration() is working?

Reply:
Do you mean writing codes like this - 
`it('value should be updated after modifying it', () => {
    const config = vscode.workspace.getConfiguration();
    // expect(config.has('salesforcedx-vscode-apex.wait-init-jobs')).toBe(true);
    config.update('salesforcedx-vscode-apex.wait-init-jobs', false);
    expect(config.get<boolean>('salesforcedx-vscode-apex.wait-init-jobs', true)).toBe(false);
  });`

I found `config.update('salesforcedx-vscode-apex.wait-init-jobs', false);` not working in this file